hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/StreamerUtil.java

```diff
@@ -255,11 +255,12 @@ public static HoodieTableMetaClient initTableIfNotExists(
           .setTableName(conf.getString(FlinkOptions.TABLE_NAME))
           .setTableVersion(conf.getInteger(FlinkOptions.WRITE_TABLE_VERSION))
           .setDatabaseName(conf.getString(FlinkOptions.DATABASE_NAME))
-          .setRecordKeyFields(conf.getString(FlinkOptions.RECORD_KEY_FIELD, null))
+          .setRecordKeyFields(conf.getString(FlinkOptions.RECORD_KEY_FIELD))
```

Comment:
Why change the default value, may not work for pk-less table scenario.

Reply:
> Why change the default value, may not work for pk-less table scenario.

Because when querying with the Log Scanner, it is necessary to merge records based on the record key